src/config/strategies/setup-user.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, test } from '@jest/globals';
+import { PassAuthUser, UserShibbolethAttrs } from '../../@types';
+
+describe('setup-user', () => {
```

Comment:
Since we have a test now, should we set up a CI.yml? Maybe run linting in it too?

Reply:
I wanted to ask you if we should create a ticket to add tests to pass-auth?  Not sure if it would be better to have jest unit tests or some kind of integration test using ts-node or docker, but seems like there should be tests for this project given it is for authentication.

Reply:
So at the moment we've got end-to-end acceptance tests which exercise pass-auth interacting with both the idp that is part of pass-docker and pass-ui.

https://github.com/eclipse-pass/pass-acceptance-testing/blob/main/tests/commonTest.js#L6-L47

Tests are always welcome, but this service has felt like a particularly tricky one to meaningfully test, and given we have the acceptance tests, I'm not sure how worth it is.

That and we've considered moving the auth service into pass-core as well.